lib/testutils/src/lib.rs

```diff
@@ -635,38 +634,20 @@ pub fn assert_abandoned_with_parent(
 }
 
 pub fn assert_no_forgotten_test_files(test_dir: &Path) {
-    // We require `taplo` for this check; if it's not installed, that's ok unless
-    // we're running in CI.
-    if !is_external_tool_installed("taplo") {
-        ensure_running_outside_ci("`taplo` must be in the PATH");
-        eprintln!(
-            "Skipping check for forgotten test files because taplo is not installed on the system"
-        );
-        return;
-    }
-
-    // Use taplo to find all the test executable main modules listed in `[[test]]`.
```

Comment:
Could you perhaps bring back a comment to explain `test_bin_mods`, mainly to tie back into the `// Add to that...` comment down below? Perhaps
```rust
// Parse the integration test binaries' main modules from the Cargo manifest.
```

Reply:
Good point, done.